hypervisor/include/arch/x86/board.h

```diff
@@ -28,9 +28,9 @@ struct vmsix_on_msi_info {
 extern struct dmar_info plat_dmar_info;
 
 #ifdef CONFIG_RDT_ENABLED
-extern struct platform_clos_info platform_l2_clos_array[MAX_PLATFORM_CLOS_NUM];
-extern struct platform_clos_info platform_l3_clos_array[MAX_PLATFORM_CLOS_NUM];
-extern struct platform_clos_info platform_mba_clos_array[MAX_PLATFORM_CLOS_NUM];
+extern struct platform_clos_info platform_l2_clos_array[HV_SUPPORTED_MAX_CLOS];
+extern struct platform_clos_info platform_l3_clos_array[HV_SUPPORTED_MAX_CLOS];
+extern struct platform_clos_info platform_mba_clos_array[HV_SUPPORTED_MAX_CLOS];
 #endif
```

Comment:
we should use MAX_CACHE_CLOS_NUM_ENTRIES for CAT array and MAX_MBA_CLOS_NUM_ENTRIES for MBA array instead of MAX_PLATFORM_CLOS_NUM. But looks like you have taken care of this in other patches.

Reply:
> we should use MAX_CACHE_CLOS_NUM_ENTRIES for CAT array and MAX_MBA_CLOS_NUM_ENTRIES for MBA array instead of MAX_PLATFORM_CLOS_NUM. But looks like you have taken care of this in other patches.

Yes, this was taken care in other patches in this PR